doc/manuals/orion-api.md

```diff
@@ -334,7 +343,15 @@ The list of builtin attributes is as follows:
 * `dateExpires` (type: `DateTime`): entity expiration date as an ISO 8601 string. How the server
   controls entity expiration is an implementation specific aspect.
 
-Like regular attributes, they can be used in `q` filters and in `orderBy`.
+* `alterationType` (type: `Text`): specifies the change that triggers the notification. It is related with 
+the subscriptions based in alteration type features (see [Subscription based in alteration type](#subscriptions_alttype) section). This attribute
+  can be used only in notifications, it does not appear when querying it (`GET /v2/entities?attrs=alterationType`) and can take the following values:
+   * `entityCreate` if the update that triggers the notification is a entity creation operation 
+   * `entityUpdate` if the update that triggers the notification was an update but it wasn't an actual change
+   * `entityChange` if the update that triggers the notification was an update with an actual change or not an actual change but with `forcedUpdate` in use
+   * `entityDelete` if the update that triggers the notification was a entity delete operation
+
+Like regular attributes, they can be used in `q` filters and in `orderBy` (`alterationType` is not included).
```

Comment:
Not sure if we should revert this way:

```suggestion
Like regular attributes, they can be used in `q` filters and in `orderBy`.
```

Why the mention to `alterationType` is needed here?

Reply:
Because in the document `ngsiv2_implementation_notes.md` file, under the section `alterationType` attribute, says:

> Apart from the attributes described in the "Builtin Attributes" section in the NGSIv2 specification,
>  Orion implements the `alterationType` attribute.

Reply:
```suggestion
Like regular attributes, they can be used in `q` filters and in `orderBy` (except `alterationType`).
```